tests/gui/test_widgets.py

```diff
@@ -2723,9 +2723,20 @@ def test_SpeechBubble_init(mocker):
     mock_download_error_connect = mocker.Mock()
     mock_download_error_signal.connect = mock_download_error_connect
 
-    sb = SpeechBubble("mock id", "hello", mock_update_signal, mock_download_error_signal, 0, 123)
+    mock_authenticated_user = factory.User()
```

Comment:
This addition wouldn't be necessary if the `authenticated_user` parameter had a default value.

Since its type is `Optional[User]` an obvious default value is `None`. (I say "obvious" in this case to highlight the fact that `None` is often an adequate default value for `Optional` types.)